pkgs/development/libraries/sentencepiece/default.nix

```
@@ -0,0 +1,29 @@
+{ config
+, stdenv
+, lib
+, fetchgit
+, cmake
+, gperftools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sentencepiece";
+  version = "0.1.84";
+
+  src = fetchgit {
```

please use fetchFromGitHub
```suggestion
  src = fetchFromGitHub {
    owner = "google";
    repo = pname;
    rev = "v${version";
    sha = ....
```